itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenarios.java

```diff
@@ -589,6 +590,72 @@ public void testIncrementalAdds() throws IOException {
   }
 
   @Test
+  public void testIncrementalLoadWithVariableLengthEventId() throws IOException, TException {
```

Comment:
truncate(event id 99) followed by 1 insert(event id 100) would be sufficient ? 

there are multiple events and iterator is from 90 might not required.